index.html

```diff
@@ -189,12 +188,13 @@ <h2 class="about-me-heading">About me</h2>
             <p class="about-me-para">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
             <button class="btn btn-resume">Get My Resume</button>
         </div>
-        <div class="about-me-image">
-            <img id="abc" src="images/Frame.png" alt="about me">
-        </div>
     </div>
     <div class="divider"></div>
+    </section>
+
+   <!-- Desktop Languages Section -->
 
+    <section class="languages-section">
```

Comment:
[ OPTIONAL]
- [ ] Kindly consider using the same color for the elements as the ones provided in Figma.
- [ ] Kindly make sure the correct spacing `margin/padding` is used to place your element on the page as it is the case Figma template.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193927871-300f8cfa-4d7b-4680-8aa1-74370606f129.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193928030-99687bf0-03f1-4a30-a5d5-c833644c931b.png)

Reply:
I have changed the color and aligned the item in the center.